tools/src/main/java/org/apache/kafka/tools/VerifiableProducer.java

```diff
@@ -241,7 +240,7 @@ public static VerifiableProducer createFromArgs(ArgumentParser parser, String[]
         } else {
             parser.printHelp();
             // Can't use `Exit.exit` here because it didn't exist until 0.11.0.0.
-            System.exit(0);
+            Exit.exit(0);
```

Comment:
Did you read the comment immediately above this line?

Reply:
Haha, no I didn't. Sorry about that.